src/deflate/zopfli_oxipng.rs

```diff
@@ -6,10 +6,10 @@ pub fn deflate(data: &[u8], iterations: NonZeroU8) -> PngResult<Vec<u8>> {
 
     let mut output = Vec::with_capacity(max(1024, data.len() / 20));
     let options = zopfli::Options {
-        iteration_count: iterations,
+        iteration_count: iterations.into(),
         ..Default::default()
     };
```

Comment:
Do you think this would be a good time to expose the entire options struct via the API? I mean, as long as we're making breaking changes 😅

Reply:
I think we should definitely explore using or exposing more Zopfli compression options at some point, but for now I think it's fine as it is. Gotta leave something for the v10 changelog too :joy:

Reply:
Ha, all good, it's just I think @Pr0methean added that new option in the hopes of using it with oxipng eventually.

Reply:
We'll get to it at some point! :tada: